model/internal/otlp_wrapper.go

```diff
@@ -148,3 +139,37 @@ func intHistogramToHistogram(src *otlpmetrics.Metric_IntHistogram) *otlpmetrics.
 		},
 	}
 }
+
+func intGaugeToGauge(src *otlpmetrics.Metric_IntGauge) *otlpmetrics.Metric_Gauge {
+	datapoints := []*otlpmetrics.NumberDataPoint{}
+	for _, datapoint := range src.IntGauge.DataPoints {
+		datapoints = append(datapoints, &otlpmetrics.NumberDataPoint{
+			Labels:            datapoint.Labels,
+			TimeUnixNano:      datapoint.TimeUnixNano,
+			StartTimeUnixNano: datapoint.StartTimeUnixNano,
+			Exemplars:         intExemplarToExemplar(datapoint.Exemplars),
+			Value:             &otlpmetrics.NumberDataPoint_AsInt{AsInt: datapoint.Value},
+		})
+	}
+	return &otlpmetrics.Metric_Gauge{
+		Gauge: &otlpmetrics.Gauge{
+			DataPoints: datapoints,
+		},
+	}
+}
+
+func intExemplarToExemplar(src []otlpmetrics.IntExemplar) []*otlpmetrics.Exemplar { //nolint:staticcheck // SA1019 ignore this!
```

Comment:
In a followup PR, we need to updated the change for the exemplars https://github.com/open-telemetry/opentelemetry-collector/pull/2402, because we don't have DoubleExemplars anymore :)

So need to change the `sed` file, and regenerate the protos, and update the pdata generator to make Exemplars array of values.

Reply:
I see where the DoubleExemplar is in the sed file, though im not seeing where that's being utilized in the protos as is. They're all either `Exemplar` or `IntExemplar`, does the `Double` get dropped somewhere?

Reply:
Ah. you're suggesting we remove them from the sed file as they're not used anymore?